### PR TITLE
fix(events): update requesting sick days with modal

### PR DESCRIPTION
**Pull request by PatrickDohn**
#### Summary

_Enumerate and describe your changes_

- update sick day requests to use modal form.

#### Checklist

_Double check list and complete these actions before merge_

- [x] PR title is in format `TYPE(scope): description`
- [x] Link PR to Issue (if any issues are related)

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -440,6 +441,63 @@ app.action(SELECT_SICK_DURATION, async (response) => {
...
+          channel: userMessage.channel,
+          ts: userMessage.ts,
+        },
+      },
+      { status: RESOLVED },
+    ),
+    ts: requestMessage.ts,
+  });
+});
+
```
Comment (PatrickDohn) [inline src/app.js:500]: Remove all extra lines

File: src/app.js [hunk 2]
```diff
@@ -746,6 +803,14 @@ app.action(OPEN_PTO_REQUEST_MODAL, async (response) => {
...
 });
 
+app.action(OPEN_SICK_REQUEST_MODAL, async (response) => {
+  await response.ack();
+  const { client, body } = response;
+  const { views } = client;
+  const view = SlackBlockKitUtil.buildSickRequestModal();
+  views.open({ view, trigger_id: body.trigger_id });
+});
+
```
Comment (PatrickDohn) [inline src/app.js:813]: Using modal form now instead of dropdown

File: src/utils/slackBlockKitUtil.js [hunk 3]
```diff
@@ -544,6 +614,23 @@ class SlackBlockKitUtil {
         };
         break;
       }
+      case RESOLVED: {
+        messageHeader = {
+          type: 'section',
+          text: {
+            type: 'mrkdwn',
+            text: `Your sick request has been submitted.`,
```
Comment (davidholyko) [inline src/utils/slackBlockKitUtil.js:622]: maybe green check mark here?

File: src/utils/slackBlockKitUtil.js [hunk 4]
```diff
@@ -667,7 +754,17 @@ class SlackBlockKitUtil {
           type: 'section',
           text: {
             type: 'mrkdwn',
-            text: `*You have a new request:*\n${name} (${email})`,
+            text: `:rotating_light: *You have a new request:*\n${name} (${email})`, // ADD EMOJIS
```
Comment (davidholyko) [inline src/utils/slackBlockKitUtil.js:757]: can remove comment?

**Review by davidholyko — APPROVED** [review]
LGTM

File: src/utils/slackBlockKitUtil.js (hunk 3 above)
Comment (PatrickDohn) [reply src/utils/slackBlockKitUtil.js]: Fixed

File: src/utils/slackBlockKitUtil.js (hunk 4 above)
Comment (PatrickDohn) [reply src/utils/slackBlockKitUtil.js]: Fixed

**Pull request merged**